packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable class-methods-use-this */
-import { ICommandInput, IFlowData } from 'amplify-cli-shared-interfaces';
-import { IFlowReport } from 'amplify-cli-shared-interfaces/lib/amplify-cli-flow-reporter-types';
+import { ICommandInput, IFlowData } from '@aws-amplify/amplify-cli-shared-interfaces';
+import { IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces/lib/amplify-cli-flow-reporter-types';
```

Comment:
we should fix this `lib` import.

Reply:
how would we fix? Leaving as it was before?

Reply:
```suggestion
import { IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces';
```